tests/integ/modin/frame/test_merge.py

```diff
@@ -483,7 +428,10 @@ def test_merge_on_index_multiindex_common_labels(left_df, right_df, how, sort):
     )
 
 
-@pytest.mark.xfail(reason="SNOW-1321662 - pandas 2.2.1 update failure", strict=True)
+@pytest.mark.xfail(
+    reason="pandas bug: AssertionError: Length of order must be same as number of levels (2), got 1",
```

Comment:
could you add a link to the pandas bug here?

Reply:
I was looking for it but couldn't find this in open issues.

Reply:
Could you file a bug report in pandas and link it here? Sometimes someone else will fix the bug, and we also have a self-contained bug report to refer to when we look at this test or behavior again.

Reply:
Done